src/index.html

```diff
@@ -14,30 +14,30 @@
     <div class="card" data-qa="card">
       <a
         href="#"
-        class="main-link"
+        class="card__link"
         data-qa="hover"
       >
```

Comment:
You don't need to put all your card content into tag **a**. Instead of this, I recommend delete tag **a** and transferring data-qa="hover" into your block card__buy.